doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -129,6 +129,13 @@ Note what happens when you try to move an end-effector out of its reachable work
 .. image:: rviz_plugin_invalid.png
    :width: 700px
 
+Moving Joints or in Null Space
+++++++++++++++++++++++++++++++++++
+You can use the **Joints** tab to control single joints and to move the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in the animation below.
```

Comment:
This tab does not *control* the joints, it can simply change individual joints of the goal state as far as I can see.

Reply:
Technically correct is the best kind of correct 🙄